src/Api/Types.hs

```diff
@@ -28,7 +28,8 @@ import           Prelude         (tail)
 
 newtype Time
     = Time Int64
-    deriving(A.ToJSON, A.FromJSON, Show, Read, Eq, Ord, Bounded)
+    deriving stock (Show, Read, Eq, Ord, Bounded)
```

Comment:
These had to be refactored after adding `DeriveAnyClass` to the default extensions (compiler errors)